app/controllers/admin/routes.py

```diff
@@ -407,6 +429,47 @@ def removeFromSession():
 
     return ""
 
+@admin_bp.route('/manageServiceLearning/imported/<courseID>', methods = ['POST', 'GET'])
+@admin_bp.route('/manageServiceLearning/imported/<courseID>', methods = ['POST'])
+def alterImportedCourse(courseID):
+    """
+    This route handles a GET and a POST request for the purpose of imported courses. 
+    The GET request provides preexisting information of an imported course in a modal. 
+    The POST request updates a specific imported course (course name, course abbreviation, 
+    hours earned on completion, list of instructors) in the database with new information 
+    coming from the imported courses modal. 
+    """
+    if request.method == 'GET':
+        try:
+            targetCourse = Course.get_by_id(courseID)
+            targetInstructors = CourseInstructor.select().where(CourseInstructor.course == targetCourse)
+            
+            courseData = model_to_dict(targetCourse, recurse=False)
+            serviceHours = (list(CourseParticipant.select()
+                                                  .where(CourseParticipant.course_id == targetCourse.id))[0]
+                                                  .hoursEarned)
+            
+            courseInstructors = [model_to_dict(instructor.user) for instructor in targetInstructors]
+            courseData['instructors'] = courseInstructors
+            courseData["hoursEarned"] = serviceHours
+
```

Comment:
remove some unnecessary lines, ones that are empty!

Reply:
Fixed